configuration/api.go

```diff
@@ -139,6 +139,76 @@ func (a *API) GetBot(id string, fields []string) (*Bot, error) {
 	return resp, err
 }
 
+// ResetBotSecret resets value of secret used to get bot token.
+func (a *API) ResetBotSecret(id, ownerClientID string) (string, error) {
+	var resp resetBotSecretResponse
+	err := a.Call("reset_bot_secret", &resetBotSecretRequest{
+		BotID:         id,
+		OwnerClientID: ownerClientID,
```

Comment:
set `OwnerClientID` similarly to how it's done in `RegisterWebhook`, applies to all methods